beacon-chain/powchain/service.go

```diff
@@ -90,8 +90,8 @@ type Web3Service struct {
 	blockCache              *blockCache // cache to store block hash/block height.
```

Comment:
Most changes in this file are just moving things around - no need for thorough review here